runelite-client/src/main/java/net/runelite/client/plugins/forn/birdhouseruns/FornBirdhouseRunsScript.java

```diff
@@ -139,17 +144,37 @@ public boolean run(FornBirdhouseRunsConfig config) {
         return true;
     }
 
+    private void emptyNests() {
+        do {
+            Rs2Inventory.interact(22798, "search");
```

Comment:
use ItemID class for magic numbers, this way if jagex ever updates the ids, it will show up in your script and you can update it correctly. Do this for the rest of the itemIds aswell if possible

Reply:
updated